courses/tests/test_signals.py

```diff
@@ -0,0 +1,40 @@
+"""
+Signal tests
+"""
+# pylint: disable=no-self-use
+import mock
+
+from django.test import TestCase
+
+from courses.factories import CourseFactory, ModuleFactory
+
+
+class PublishOnUpdateTests(TestCase):
+    """
+    Tests for webhook posting
+    """
+    def test_course_save_publishes(self):
+        """
+        When course is saved, it should trigger webhook.
+        """
+        c = CourseFactory.build()
+        with mock.patch('courses.signals.publish_webhook') as wh_mock:
+            c.save()
+            assert wh_mock.delay.call_count == 1
```

Comment:
Modules also create courses which was affecting my count. Possible that code changed since writing that to obviate it. I'll check.